ui/views.py

```diff
@@ -172,7 +172,9 @@ def get_vocabularies(facets):
         vocab = (vocabulary_id, vocabs[int(vocabulary_id)])
         vocabularies[vocab] = []
         for t_id, count in term_data:
```

Comment:
`term_data` comes from elasticsearch. I'm thinking that the delete completed before elasticsearch had a chance to be updated. Can you confirm @ShawnMilo?

I think you should leave this piece of code alone because there are better places to fix this, maybe by waiting for elasticsearch at the point of the API call.